index.html

```diff
@@ -239,10 +240,13 @@ <h2>More information <span>comming soon!</span></h2>
               <iframe src="https://www.youtube.com/embed/_Th1GTJmL2Y" allowfullscreen></iframe>
             </div>
           </div>
+        </div>
       </div>
     </div>
     <!-- Javascript includes -->
     <script src="/dist/scripts.js" charset="utf-8"></script>
+
     <!-- end includes -->
+    <iframe src="/" name="MailChimpSubscribe" id="MailChimpSubscribe" style="width: 0; height: 0; border: 0;"></iframe>
```

Comment:
wellicht een beter idee om de pagina niet in een nested iframes in te laden:p
 `src='about:blank'`